### PR TITLE
Add Macromill and Yahoo! JAPAN Ads, Tag Manager

### PR DESCRIPTION
**Pull request by sisidovski**
Hi!

I noticed `macromill.com` is treated as Yahoo! JAPAN's service, but actually Macromill is a marketing company, it's not related to Yahoo! JAPAN. In addition to that, today Yahoo! JAPAN is an independent company from Yahoo! US ([wikipedia](https://en.wikipedia.org/wiki/Yahoo!_Japan)), they're running their own businesses. So I added ads and tag manager both provided by Yahoo! JAPAN to capture those 3Ps more precisely.

### CODE REVIEW

File: data/entities.js [hunk 1]
```diff
@@ -6077,6 +6076,14 @@ module.exports = [
     domains: ['*.mlveda.com'],
     examples: ['www.mlveda.com'],
   },
+  {
+    name: 'Macromill',
+    company: 'Macromill',
+    categories: ['analytics'],
+    homepage: 'https://group.macromill.com/',
+    domains: ['*.macromill.com'],
+    examples: ['jp.moony.com/ja/home.html']
```
Comment (patrickhulce) [inline data/entities.js:6085]:
```suggestion
    examples: ['img.macromill.com/js/us000131vfg/4000000570-56/lognos.js']
```

File: data/entities.js [hunk 2]
```diff
@@ -10198,10 +10205,20 @@ module.exports = [
...
   {
-    name: 'Yahoo! Japan',
-    company: 'Yahoo!',
+    name: 'Yahoo! JAPAN Ads',
+    company: 'Yahoo! JAPAN',
     categories: ['ad'],
-    domains: ['*.macromill.com'],
+    homepage: 'https://global-marketing.yahoo.co.jp/',
+    domains: ['*.yimg.jp'],
+    examples: ['shopping.yahoo.co.jp'],
```
Comment (patrickhulce) [inline data/entities.js:10213]:
```suggestion
    examples: ['s.yimg.jp/images/ymstore/bnr/shp/paypay01/180_70.png'],
```

File: data/entities.js [hunk 3]
```diff
@@ -10198,10 +10205,20 @@ module.exports = [
...
+  },
+  {
+    name: 'Yahoo! Tag Manager',
+    company: 'Yahoo! JAPAN',
+    categories: ['tag-manager'],
+    homepage: 'https://marketing.yahoo.co.jp/service/tagmanager/',
+    domains: ['*.yjtag.jp'],
+    examples: ['www.yahoo.co.jp']
```
Comment (patrickhulce) [inline data/entities.js:10221]:
```suggestion
    examples: ['yjtag.yahoo.co.jp/tag?site=2wzBV9u']
```

**Review by patrickhulce — CHANGES_REQUESTED** [review]
thanks very much @sisidovski this is great!

the examples are for examples of specific resources, not sites that use them :) but otherwise LGTM

**Comment by sisidovski** [comment]
@patrickhulce Thank you so much! I changed example resources to the specific ones!

**Review by patrickhulce — APPROVED** [review]
thank you! 🎉

**Comment by patrickhulce** [comment]
oh shoot sorry @sisidovski you'll need to update the snapshots for the tests :)

**Comment by sisidovski** [comment]
@patrickhulce Updated!

**Pull request merged**